docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -72,14 +87,54 @@ the sample results by providing a valid interval for your data. The generated
 preview contains the number of potentially created alerts during the relative 
 time range you defined.
 
+As the last step in the rule creation process, 
+<<defining-actions, define the actions>> to take in case the configured criteria 
+are met.
+
+
+[[creating-anomaly-jobs-health-rules]]
+=== {anomaly-jobs-cap} health
+
+Select the {anomaly-job} or the group of {anomaly-jobs} that is checked against 
+the rule. If you assign additional jobs to the group, the new jobs are 
```

Comment:
Great you've mentioned that! Please also add the same for the special selector `*` that allows monitoring of all existing jobs, even those created _after_ the rule instance creation.

Reply:
Added in b25c915.